gradle.properties

```diff
@@ -6,6 +6,7 @@ kotlin_version=1.3.61
 kotlin_plugin_version=1.3.61-release-180
 idea_version=192.5728.98
 language_version=1.3
+org.gradle.jvmargs=-Xmx2g
```

Comment:
Do we need this? If so, it should be at the bottom with a new comment

Reply:
We needed this before we could change CI config.